app/controllers/report_controller/menus.rb

```diff
@@ -646,9 +646,9 @@ def menu_get_all
     roles.sort_by { |a| a.name.downcase }.each do |r|
       @sb[:menu][r.id] = r.name
     end
-    @right_cell_text = title == "My #{ui_lookup(:model => "MiqGroup")}" ?
+    @right_cell_text = title == _("My EVM Group") ?
```

Comment:
`title` always comse from `get_group_roles`, so..

Wondering if we should rewrite this to something like..

```
@right_cell_text = if super_admin_user?
  _("My EVM Group")
else
  _("All EVM Groups")
end
```

to simplify ... but that actually exactly matches what `get_group_roles` already does.

So.. this is just `@right_cell_text = title`, isn't it?

Reply:
@himdel Yep